README.md

```diff
@@ -16,32 +16,30 @@ The test framework is jest; functional areas are separated into packages and man
   - You can then use the [explorer](https://play.dgraph.io/?latest) (server URL is http://localhost:8080)
 - Initialise DGraph schema and load data into DGraph:
   - `yarn load-data`
-- Start the data-service:
-  - `yarn start-data-service`
-  - Access v1 api at http://localhost:4000/v1/graphql
-  - Access v2 api at http://localhost:4000/v2/graphql
+- Start the backend:
+  - `yarn start-backend`
+  - Access v1 api at http://localhost:4007/v1/graphql
+  - Access v2 api at http://localhost:4007/graphql
```

Comment:
```suggestion
  - Access internal api at http://localhost:4007/graphql
```

I'm thinking of the `/graphql` endpoint as something we can iterate on and use in the frontend, not a published API for external users...